entities/library.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require './entities/book'
+require './entities/reader'
+require './entities/author'
+require './entities/order.rb'
+require './seed'
+
+# library information
+class Library
+  attr_reader :readers, :orders, :authors, :books
+
+  include SeedData
+
+  def initialize
```

Comment:
`def initialize(authors: [], books: [], readers: [], orders: [])`?
no seed data in entities
You should have seeds.rb file with seed data. The seed data from this file should be loaded into the library

Reply:
done